paper.tex

```diff
@@ -22,46 +26,256 @@
-develop and refine ancestry inference methods, assessing the relative strengths
-and weaknesses of the various approaches. Using the blanket term ARG hides the
-important differences in the approaches, and hampers our ability to compare and
-improve inference methods. In this paper we discuss the different approaches we
-may use to encode genetic ancestry, and of the differing amounts of information
-about the ancestral process that we can represent and hope to observe. We
-classify existing inference methods according to the type of ancestry they
-infer, and discuss the strengths and weaknesses of the inferred structure for
-downstream applications.
+recombination at scale for the first time, enabling many
+downstream applications in population and statistical genetics.
+Such recombinant genetic ancestry is usually
+referred to as an Ancestral Recombination Graph, or ARG.
+% Note: slight repetition here with first sentence.
+There are now multiple methods that can infer ARGs at a practical scale
+and it is therefore vital that these methods can be systematically evaluated
+and compared to determine their strengths and weaknesses.
+Unfortunately, however, there is substantial confusion over
+terminology and little agreement on shared standards for data interchange,
+significantly hampering progress in this important area.
+This confusion and lack of agreement on basic issues such as what does and
+does not constitute an ARG is partly attributable to the historical
+development of the term.
+Originally rigorously defined as
+a representation of the coalescent with recombination
+stochastic process as a graph, an ARG is now understood to refer to any
+concrete realisation of a recombinant genetic ancestry.
+We show that the standard description of an ARG
+in terms of common ancestor and recombination events
+(inherited from the original stochastic process definition) is
+ambiguous, fundamentally limited and an unsuitable basis for data interchange.
+We provide a simple alternative definition of an ARG in which a node
+corresponds to one of an individuals monoploid genomes, an edge
+% jk-note: this is clumsy
+defines a genetic ancestry relationship between two nodes,
+and annotations on those edges define the precise patterns of
+ancestry in intervals along the genome.
+We use this framework to define some classifications and metrics of ARG nodes,
+and discuss the properties of some recently developed inference methods.
+% jk-note: this ends a bit flat - we should say other stuff like we clarify
+% how the ARG relates the pedigree, etc, etc. Let's go through again when
+% we have a reasonable draft.
 \end{abstract}
 
 \textbf{Keywords:} Ancestral Recombination Graphs
 
 \section*{Introduction}
+Inferring the genetic relationships between sampled genomes in the form of an
+evolutionary tree is a necessary prerequisite for many analyses of non (or
+rarely) recombining organisms [refs]. The trees capture what is known and
```

Comment:
Minor nit-pick, but should "The trees capture" be the singular "The tree captures" here, to avoid any insinuation of multiple trees for the same non-recombining sample?

Reply:
Sure, lets go with that